### DISCUSSION TITLE
【求助】制作了一个简易的fairy，但是有很多方面等待完善

### DISCUSSION THREAD

**Discussion by xrtcfbtrsh** [share121/inter-knot#1366] [其他]
![background](https://github.com/user-attachments/assets/4706aaec-1383-47bd-a515-7b11dca88b8a)
写了一个类似fairy的悬浮球，但是不知道如何打包成插件，可否有绳匠帮忙？出0.1145141919810菲林
仓库：https://github.com/xrtcfbtrsh/fairy-

**Comment by richi-shek** [comment]
这个委托我接了
仓库：[https://github.com/richi-shek/fairy-](https://github.com/richi-shek/fairy-)
另外，我提交了一个拉取请求，你可以接受拉取请求，并将它合并到你的主分支。

**Comment by capacitor1** [comment]
🤔我可以提供一种方向，但是我没时间去做

用live2d，就是很多个人博客右下角那个可以互动的东西
